charts/delphai-deployment/templates/pod-monitor.yaml

```
@@ -0,0 +1,14 @@
+{{ if $.Values.monitoring.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: "{{ $.Release.Name }}-pod-monitor"
```

Why do you need a suffix here? It's already has it's own kind
```suggestion
  name: "{{ $.Release.Name }}"
```
Also appending release name could lead to new maxlength issues